pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
removing the TLS data in conversion means we can't catch it in validation. making an object in this state fail validation would mean it couldn't be updated in etcd without setting the termination type or removing the cert/key data... what would that break?